lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMFMA.cpp

```diff
@@ -37,38 +37,34 @@ namespace SharedToDotOperandMFMA {
 // Computes offsets for operand A or transposed operand B
 // @param rewriter
 // @param loc
-// @param elemsPerThread operand tile shape consumed by one MFMA instruction
+// @param elemsPerInstr operand tile shape consumed by one MFMA instruction
 // @param waveM wave id for the "non K" axis
```

Comment:
Seems this line should be fixed too, the new parameter is `waveId`

Reply:
I'll fix it when rebase it on top of github action fix.
I hope this PR will fix it: #230

Reply:
@jayfurmanek 
I missed when you updated this branch =(
Can I add this fix with some later PR to not rerun tests?

Reply:
oh sorry, I should have left that to you. Adding the comment fix later is fine

Reply:
Fixed in https://github.com/ROCmSoftwarePlatform/triton/pull/241/files